common/data_refinery_common/utils.py

```diff
@@ -12,6 +12,7 @@
 from django.conf import settings
 from django.core.exceptions import ImproperlyConfigured
 from retrying import retry
+from multiprocessing import current_process
```

Comment:
unused import?

Reply:
I saw an error that it wasn't being imported or defined here, so I imported it.. https://github.com/AlexsLemonade/refinebio/pull/2015/files/9f39a868e98b2cb52000b5ad4f9fe8d67ddc67a3#diff-851165ed4bf22b784eb9cf6b338d69ecR66

Reply:
gotcha - i missed current_process being used there 👍